wurst/closures/ClosureFrames.wurst

```diff
@@ -0,0 +1,121 @@
+package ClosureFrames
+import NoWurst
+import Trigger
+import HashMap
+import Unit
+
+/*  
+    Example usage:
+        createSimpleFrame("MySimpleButton", GAME_UI, 0)
+        ..onClick( () -> print("The button was clicked.") )
+        ..onEnter( () -> print("The mouse is on the button.") )
+        ..onLeave( () -> print("The mouse left the button.") )
+
+    Important:
+        * Most frametypes don't support only a few events (some might not support any)
```

Comment:
> don't support only a few events

sounds confusing. Do they only support a few, or do a few not work? Judging from the statement in the bracket I would think that "Most frametypes don't support most events"

Reply:
Yeah I mixed two phrases there..